Specs/DateTime/Dutch/DateTimeModel.json

```diff
@@ -1908,27 +1900,24 @@
     "Context": {
       "ReferenceDateTime": "2016-11-07T16:12:00"
     },
-    "NotSupported": "dotnet",
     "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
-        "Text": "4:00 tot 7",
```

Comment:
@iMicknl, what's your opinion on this one?

Reply:
(note the comment about this case in the PR description)

Reply:
>"Ik ben 4:00 tot 7 weg" -> "Ik ben weg 4:00 tot 7" (I'll be out 4:00 to 7 oclock). Patters like "4:00 tot 7" that end with a pure number are extracted only if they occur at the end of the sentence.

I am not sure if I understand why it would work at the end of the sentence, but not in the middle. However, it seems like an edge case where you use `4:00 to 7` instead of `4 to 7` or `4:00 to 7:00`.

Reply:
I think to avoid extracting false patterns (this behavior is implemented in BaseTimePeriodExtractor where a comment says: "cases like "from 3:30 to 4 people" shuold not be extracted").

Reply:
@LionbridgeCSII, you are right. I forgot this durig the review. nothing to change for now.